dataland-frontend/src/components/pages/SearchCompaniesForFrameworkData.vue

```diff
@@ -158,12 +146,11 @@ export default {
       if (filtered) {
```

Comment:
On second thought: Da currentSearchbarInput und currentFilteredFrameworks standardmäßig sowiso undefined sind, und nur geändert werden, wenn nötig, könnte man die ganze filtered-Variable (wahrscheinlich) oder einfach weglassen und immer `this.$refs.frameworkDataSearchBar.queryCompany(this.currentSearchBarInput, this.currentFilteredFrameworks)` aufrufen oder?